homeassistant/components/volumio/browse_media.py

```diff
@@ -100,11 +101,13 @@ def _list_payload(media_library, item, children=None):
     )
 
 
-def _raw_item_payload(media_library, item, parent_item=None, title=None, info=None):
+def _raw_item_payload(entity, item, parent_item=None, title=None, info=None):
     if "type" in item:
         thumbnail = item.get("albumart")
         if thumbnail:
-            thumbnail = media_library.canonic_url(thumbnail)
+            item_hash = hashlib.md5(thumbnail.encode("utf-8")).hexdigest()
```

Comment:
Do we need a cryptographic hash here?  Can you use the built-in hash function instead here since its orders of magnitude faster?

Reply:
Good point. Done

Reply:
I just hope collisions aren't common...